packages/E2ETest/windows/ReactUWPTestApp/Assets/TreeDump/ControlStyleRegularBorder.txt

```diff
@@ -656,7 +656,7 @@
           HorizontalAlignment=Left
           Margin=0,0,0,0
           Padding=12,0,0,2
-          RenderSize=90,21
+          RenderSize=90,20.66667
```

Comment:
>20.66667 [](start = 24, length = 8)

if this is a master we have to make sure we aren't going to have floating point issues with these values...can we force this back into integer values?

Reply:
Yeah, I'm not sure what's going on here. I'm going to revert the changes to the masters and go through the CI logs again.